disasm/disasm.cc

```diff
@@ -2233,38 +2233,43 @@ void disassembler_t::add_instructions(const isa_parser_t* isa)
     DEFINE_R1TYPE(sm3p1);
   }
 
-  if (isa->extension_enabled(EXT_ZVBB)) {
+  if (isa->extension_enabled(EXT_ZVKB) || isa->extension_enabled(EXT_ZVBB)) {
```

Comment:
Not sure if this is the accepted way to condition instructions shared between two extensions.

Reply:
You only need to check for Zvkb because you added this line, which guarantees that if Zvbb is present then Zvkb is also present.

https://github.com/riscv-software-src/riscv-isa-sim/pull/1474/files#diff-28b412bc112a3a620c001f69229c74da5d1cdf37a0293562e9209ee93c6dff04R240